lms/resources/_js_config/__init__.py

```diff
@@ -453,9 +453,13 @@ def _get_product_info(self):
             product_info["api"]["listGroupSets"] = {
                 "authUrl": self._request.route_url(product.route.oauth2_authorize),
                 "path": self._request.route_path(
-                    product.route.list_group_sets,
-                    course_id=api_course_id,
+                    "api.courses.group_sets.list", course_id=api_course_id
                 ),
+                "data": {
```

Comment:
We need to explicitly pass the product family back to ourselves.

In the context of the API it might be impossible to know in which LMS we operate.

But we do know that in the launch, so just send that back to us.

This mechanism is not new, just new for this view now.

Reply:
Can't we tell from the authentication of the API request which LMS we're operating in? The authenticated user should be associated with an application instance

Reply:
No, unfortunately we can tell 100% of the time which product a given application instance belongs to. It's probably very close to that 100% thought.

We probably should start logging the edge cases and get a sense of the actual numbers.